test/unitTests/OmnisharpDownloader.test.ts

```diff
@@ -59,14 +59,16 @@ suite('OmnisharpDownloader', () => {
     });
 
     test('Events are created', async () => { 
```

Comment:
Test for the "bad download -> good download" and "bad download -> bad download>" cases?

Reply:
Added the test in the download validation